tests/flit_cfg.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Load a pre-trained Caffe2 image classifier and run it on an image.
+
+# Configuration file for the 'lit' test runner.
+
+import os
+
+def getToolPath(tool_name, tools_dir):
+    tool_path = os.path.normpath(os.path.join(tools_dir, tool_name))
+    if not os.path.exists(tool_path):
+        return None
+    return tool_path
+
+def setup_config(config):
+    # name: The name of this test suite.
+    config.name = 'glow'
+
+    # suffixes: A list of file extensions to treat as test files.
+    config.suffixes = [".test"]
+
+    # Set the different string substitutions that we use in our test suite.
+    config.tool_to_path = {}
+
+    for tool_name in ('text-translator', 'image-classifier', 'model-runner'):
```

Comment:
I am not sure why we need this logic. Can we just pick up the tools from the path?  Listing all of the tools manually just makes it difficult to add new tools. Can we get rid of this file?